doc/reference/configuration/index.rst

```diff
@@ -296,66 +300,106 @@ dynamic, that is, they can be changed at runtime by calling ``box.cfg{}``
 a second time.
 
 To see all the non-null parameters, say ``box.cfg`` (no parentheses). To see a
-particular parameter, for example the listen address, say ``box.cfg.listen``.
+particular parameter, for example, the listen address, say ``box.cfg.listen``.
+
+..  _box-cfg-params-prior:
+
+Methods of setting and priorities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tarantool configuration parameters can be specified in different ways.
+The priority of parameter sources is the following, from higher to lower:
+
+*   ``box.cfg{}`` call
+*   :ref:`environment variables <box-cfg-params-env>`
+*   :doc:`tarantoolctl </reference/tarantoolctl>` options
+*   default values.
+
+..  _box-cfg-params-env:
+
+Setting via environment variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting from version :doc:`2.8.1 </release/2.8.1>`, you can specify configuration parameters via special environment variables.
+The name of a variable should have the following pattern: ``TT_<NAME>``,
+where ``<NAME>`` is the uppercase name of the corresponding :ref:`box.cfg parameter <box-cfg-params-ref>`.
+
+For example:
+
+* ``TT_LISTEN``---corresponds to the ``box.cfg.listen`` option.
+* ``TT_REPLICATION``---corresponds to the ``box.cfg.replication`` option.
```

Comment:
It worth to add one example of an option, whose name is constructed from several words.

Reply:
Added ``memtx_dir`` as an example of such options.